config/201-channelable-manipulator-clusterrole.yaml

```diff
@@ -17,7 +17,7 @@
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: events-channel-addressable-resolver
+  name: events-channel-manipulator-resolver
```

Comment:
I think this should be `events-channel-channelable-manipulator` to be consistent with https://github.com/knative/eventing/blob/fa35b5fb53a1e4c9b1ae99cda4874cabbebc2f02/config/channels/in-memory-channel/200-channelable-manipulator-clusterrole.yaml#L18 and https://github.com/knative/eventing-contrib/blob/5408c00d064038924c628b784562f52b32dad101/kafka/channel/config/200-channelable-manipulator-clusterrole.yaml#L18